### ISSUE TITLE
service send log rule but in windows event logger don't have any trace of this rule

### ISSUE THREAD

**Issue by KalvinVilla** [SA-EME/SWA#9]
```
[INFO] 23/04/2024 08:47:32 +02:00 WFRAMPB002 Microsoft-Windows-Security-Auditing Warning 4625 Echec de connexion  Nom du compte :  villakl-adm  Raison de l’échec :  %%2313 depuis  Adresse du réseau source 
[INFO] 23/04/2024 16:23:04 +02:00 WFRAMPB002 Microsoft-Windows-Security-Auditing Warning 4625 Echec de connexion  Nom du compte :  villakl-adm  Raison de l’échec :  %%2313 depuis  Adresse du réseau source 
[INFO] Initialisation du service SWA 0.1.1
[INFO] Importation du fichier C:\Program Files\SWA\Templates\001-fr_failed_auth.swr
[INFO] Ajout de la règle 001-fr_failed_auth.swr à l'écouteur Security
```

**Comment by KalvinVilla** [comment]
try to add log in send func.

**Comment by KalvinVilla** [comment]
fix by [380ccdf](https://github.com/SA-EME/SWA/commit/380ccdfba14b798d8712247e20ac6ef724dbe58c)
